### ISSUE TITLE
Firth penalized likelihood function in assocCoxPH

### ISSUE THREAD

**Issue by ryshi06** [smgogarten/GWASTools#8]
Hi there, 

I am wondering is assocCoxph support firth's penalized likelihood function or other possible functions adjusting for monotone likelihood issues?

Thanks a lot!

Best,
Ruyu

**Comment by smgogarten** [comment]
No, it does not. While assocRegression has an option to use the Firth test (via the [logistf](https://cran.r-project.org/web/packages/logistf/) package), assocCoxPH does not have that option. There is a package [coxphf](https://cran.r-project.org/web/packages/coxphf/) that would provide the test you're looking for, but I didn't know it existed until I [looked it up](https://cemsiis.meduniwien.ac.at/en/kb/science-research/software/statistical-software/firth-correction/) just now to respond to your question. 

This is not a feature I will have time to add anytime soon, but if you're interested in adding it to GWASTools yourself, you could submit a pull request.

**Comment by ryshi06** [comment]
Thanks for your reply. I 'll try the assocCoxPH first to decide whether I do need firth's correction. Thanks again!

Best,
Ruyu
